form.js

```diff
@@ -15,3 +17,31 @@ form.addEventListener('submit', (e) => {
     form.submit();
   }
 });
+
+let localData = JSON.parse(localStorage.getItem('userInput'));
+let formInput = {
+  name: document.getElementById('name').value,
+  email: document.getElementById('email').value,
+  message: document.getElementById('message').value,
+};
+localStorage.setItem('userInput', JSON.stringify(formInput));
+
+const isInputChange = (input) => {
+  input.addEventListener('change', (e) => {
+    formInput = {
+      name: document.getElementById('name').value,
+      email: document.getElementById('email').value,
+      message: document.getElementById('message').value,
+    };
+    formInput[input.name] = e.target.value;
+    localStorage.setItem('userInput', JSON.stringify(formInput));
+    localData = JSON.parse(localStorage.getItem('userInput'));
+  });
+};
+
+nameValue.value = localData.name;
+emailValue.value = localData.email;
+messageValue.value = localData.message;
+isInputChange(nameValue);
+isInputChange(emailValue);
+isInputChange(messageValue);
```

Comment:
The form must be populated on page load with the data stored in the localStorage. Currently, when I reload the page the data is cleared in the localStorage as demonstrated below.

![image](https://user-images.githubusercontent.com/38088818/173597329-5d6ef298-1a66-4191-af0f-205459b812ff.png)

- I can see that when I perform the first reload the data still reflects on the form although the localStorage is cleared. Then, when I refresh the second time the values disappear.
- Kindly look into this and ensure that the data is preserved on the localStorage and the form is populated with the data kept in the local storage on page load.